example/newreposecret/main.go

```diff
@@ -0,0 +1,179 @@
+// Copyright 2018 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// newreposecret creates a new secret in GitHub for a given owner/repo
+// It has two required flags for owner and repo, and takes in one argument for the name of the secret to add.
+// Provide the value of the secret you want to add with an environment variable of the same name.
+// To authenticate with GitHub provide it via an environment variable GITHUB_AUTH_TOKEN
+//
+// To verify the new secret navigate to GitHub Repository > Settings > left side options bar > Secrets
+//
+// Usage:
+//	export GITHUB_AUTH_TOKEN=<auth token from github that has secret create rights>
+//	export SECRET_VARIABLE=<secret value of the secret variable>
+//	go run main.go -owner <owner name> -repo <repository name> SECRET_VARIABLE
+//
+// Example:
+//	export GITHUB_AUTH_TOKEN=0000000000000000
+//	export SECRET_VARIABLE="my-secret"
+//	go run main.go -owner google -repo go-github SECRET_VARIABLE
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"flag"
+	"fmt"
+	sodium "github.com/GoKillers/libsodium-go/cryptobox"
```

Comment:
Did you run this code through `gofmt`?  I'm surprised the GitHub workflow tests did not catch this!  In fact, I'm going to stop this code review here and give you my feedback so far and see if I can fix this.

Just FYI... I use Emacs primarily, vim sometimes, and VSCode every now and then... all are great editors, and the last one has amazing Go support.  Whatever editor(s) you use, you should have them set to "format on save" or run "goimports" to update your imports and also fix the formatting... it's a huge time-saver!

Reply:
OK, Today I Learned that "goimports" is the one that nicely splits up imports into standard library and external ones with a blank line in between.  I thought that was "gofmt", but I was wrong.  If you could please add a blank line between standard library imports and external imports, or just use "goimports", that would be great, so that our repo is consistent... thank you!

Reply:
I had never heard of goimports, so I added that to my idea (Jetbrains Goland) so it should catch this in the future!